torch_xla/csrc/debug_util.cpp

```diff
@@ -116,8 +116,8 @@ void DebugUtil::SaveTensorsGraphInfo(const char* name,
                                      absl::Span<const XLATensorPtr> tensors,
                                      const std::vector<size_t>* indices,
                                      GraphFormat format) {
-  static const std::string save_file =
-      xla::sys_util::GetEnvOrdinalPath("XLA_SAVE_TENSORS_FILE", "");
+  thread_local const std::string save_file = xla::sys_util::GetEnvOrdinalPath(
```

Comment:
haha I need to google first to confirm that `thread_local` is implicitly static.

Reply:
Same! https://stackoverflow.com/a/22794640